test/format/format_config.c

```diff
@@ -1152,18 +1152,17 @@ config_mirrors(void)
 {
     u_int available_tables, i, mirrors;
     char buf[100];
-    bool already_set, explicit_mirror, fix, var;
+    bool already_set, explicit_mirror;
 
-    fix = var = false;
-    g.mirror_fix_var = false;
+    g.mirror_col_store = false;
     /* Check for a CONFIG file that's already set up for mirroring. */
     for (already_set = false, i = 1; i <= ntables; ++i)
         if (NTV(tables[i], RUNS_MIRROR)) {
             already_set = tables[i]->mirror = true;
             if (tables[i]->type == FIX)
-                fix = true;
+                g.mirror_col_store = true;
             if (tables[i]->type == VAR)
-                var = true;
+                g.mirror_col_store = true;
```

Comment:
Minor: Here and below these two conditionals could be combined to be:
if (tables[i]->type == FIX || tables[i]->type == VAR)
  g.mirror_col_store = true;

Reply:
[1fae2ae](https://github.com/wiredtiger/wiredtiger/pull/9523/commits/1fae2ae35a8153beb0287889bda3ee908c7f3612)